LayoutTests/fast/canvas/CanvasRenderingContext2DSettings-colorSpace-enabled.html

```diff
@@ -9,7 +9,7 @@
         let canvas = document.createElement("canvas");
         let context = canvas.getContext("2d", settings);
         window.testSettings = context.getContextAttributes();
-        shouldBeEqualToString("testSettings.colorSpace", "srgb");
+        shouldBeEqualToString("testSettings.colorSpace", expectedColorSpace);
     }
```

Comment:
Can we add a test case for "display-p3" for the enabled case like what we did for the disabled case?
```
// Test setting colorSpace to "display-p3" works.
testColorSpace({ colorSpace: "display-p3" }, "display-p3");
```

Reply:
Thanks @mattwoodrow for suggesting the Internals test.
I went with a single test file + expected, because display-p3 support is not as simple as `platform==Cocoa?`.